flowetl/etl/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="etl",
+    url="https://github.com/Flowminder/FlowKit",
+    maintainer="Flowminder",
+    packages=find_packages(),
+    include_package_data=True,
+    zip_safe=False,
+    install_requires=["apache-airflow==1.10.3"],
```

Comment:
Do I remember correctly that you said the pinned version is to ensure that it's the same version as inside the docker container? Can we add a comment here reminding us of this, so that we can update it as and when the airflow docker images gets updated?

Reply:
Yeah